Client/Frontend/Strings.swift

```diff
@@ -264,6 +264,13 @@ extension Strings {
     public static let ContextMenuButtonToastNewPrivateTabOpenedButtonText = NSLocalizedString("ContextMenu.ButtonToast.NewPrivateTabOpened.ButtonText", value: "Switch", comment: "The button text in the Button Toast for switching to a fresh New Private Tab.")
 }
 
+// UI Notification Settings, for push
+extension Strings {
+    public static let SentTabViewActionTitle = NSLocalizedString("View", comment: "View a URL - https://bugzilla.mozilla.org/attachment.cgi?id=8624438, https://bug1157303.bugzilla.mozilla.org/attachment.cgi?id=8624440")
+    public static let SentTabBookmarkActionTitle = NSLocalizedString("Bookmark", comment: "Bookmark a URL - https://bugzilla.mozilla.org/attachment.cgi?id=8624438, https://bug1157303.bugzilla.mozilla.org/attachment.cgi?id=8624440")
+    public static let SentTabAddToReadingListActionTitle = NSLocalizedString("Add to Reading List", comment: "Add URL to the reading list - https://bugzilla.mozilla.org/attachment.cgi?id=8624438, https://bug1157303.bugzilla.mozilla.org/attachment.cgi?id=8624440")
```

Comment:
Nit: Probably want to prefix the string identifier with `SentTab.*` to help disambiguate. Also don't need the bug - maybe a more descriptive sentence about what this string is used for?